Automattic/wp-calypso
# Photos picker: Media library: Update Google Photos icon

Related to pdtkmj-36s-p2#comment-5914

## Proposed Changes

* Add Google Photos icon

## Why are these changes being made?


* Replace generic gridicon

## Testing Instructions



* Go to `/media/{SITE_SLUG}`
* Click on external source dropdown
* Check if there is a google photos icon

| Before | After |
|--------|--------|
| <img width="242" alt="Screenshot 2024-11-28 at 11 03 32" src="https://github.com/user-attachments/assets/2f7699d7-5ad6-41d4-949d-aeb8441bda71"> | <img width="253" alt="Screenshot 2024-11-28 at 11 03 05" src="https://github.com/user-attachments/assets/dde81c24-15ac-4827-88ad-9a2437acc294"> |
| <img width="240" alt="Screenshot 2024-11-28 at 11 03 39" src="https://github.com/user-attachments/assets/85c8b0c5-aa9a-4be2-bc05-cb27c3a2314f"> | <img width="246" alt="Screenshot 2024-11-28 at 11 03 11" src="https://github.com/user-attachments/assets/7436717e-5399-44e9-abfc-9701517750d7"> | 

## Pre-merge Checklist
